RFC-by-stage/1-approved/monorepo.md

```diff
@@ -0,0 +1,19 @@
+---
+Status: `Proposal` # Please do not change this.
+Implementer: # It will be changed upon merging and as it moves through the RFC stages
+---
+
+# Simplify the monorepo
+
+## The issue to be solved
+
+The tool [lerna](https://lerna.js.org/) we have been using to power the monorepo has become incredibly slow and difficult to deal with.
+We need to simplify the structure to allow us to fix things faster.
+
+## A short description of the solution
+
+I propose we remove lerna, move to [yarn v1 workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) and de-tangle the reasons the `bootstrap` process is currently so dependent on a very specific order at which what dependency has to be installed.
+
+## Technical details
+
+TODO 😬
```

Comment:
On assumption to remain as multi-repo: 

1. Remove Lerna
2. Remove custom WebPack configuration scripts
3. Remove test infrastructure
4. New directory structure for packages
5. New monorepo tooling
6. New Storybook based development environment - using tests/site content
7. New unit tests tooling 
8. New styles tooling 
9. New Vanilla js tooling 
10. New React.js tooling

Reply:
This RFC does not talk about implementation of any third party testing app like storybook.
A lot of those things in your list are way out of scope.
We won't need to touch the testing luckily and storybook is not the right tool for this as it will bloat the entire thing and this was about reducing things. It's also not a great tool when it comes to a11y for designers and others.
Styles can we reused as they are now and we can look into making them better in another RFC.

Reply:
Acknowledged, resolving